pkg/accounting/tally/stats.go

```diff
@@ -0,0 +1,84 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package tally
+
+import (
+	"storj.io/storj/pkg/pb"
+)
+
+//Stats all stats fields
+type Stats struct {
```

Comment:
This probably doesn't need to be public.

Reply:
I thought about it.. but I am guessing going forward we might move this file out to Pkg,  just to handle more statistics. Should we have a separate pkg for statistics?

Reply:
no this is pretty tally specific. i agree it doesn't need to be public

Reply:
making it private